lesson10/lesson10.js

```diff
@@ -0,0 +1,57 @@
+const ul = document.getElementById('js-list');
+const listItems = [
+  { to: "bookmark.html", img: "bookmark-icon.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "mail-icon.png", alt: "画像2", text: "メッセージ" }
+];
+
+function showLoading() {
+  const loadingImg = document.createElement('img');
+  loadingImg.src = "loading-circle.gif";
+  loadingImg.id = "loading-img";
+  ul.appendChild(loadingImg);
+}
+
+function removeLoading() {
+  const loadingID = document.getElementById('loading-img');
+  loadingID.remove();
+}
+
+function createList(item) {
+  const fragment = document.createDocumentFragment();
+  for (let i = 0; i <= item.length - 1; i++) {
+    const li = document.createElement('li');
+    const a = document.createElement('a');
+    const img = document.createElement('img');
+    a.textContent = item[i].text;
+    a.href = item[i].to;
+    img.src = item[i].img;
+    img.alt = item[i].alt;
+    fragment.appendChild(li).appendChild(a).appendChild(img);
+  }
+  ul.appendChild(fragment)
+}
+
+function fetchListData() {
+  const promise = new Promise((resolve) => {
+    setTimeout(() => resolve(listItems), 3000)
+  })
+  return promise;
+}
```

Comment:
```suggestion
function fetchListData() {
   return  new Promise((resolve) => {
    setTimeout(() => resolve(listItems), 3000)
  })
}
```

nits
You can also write something like this

Reply:
Thank you for your comment! Thanks to you, we were able to simplify the inside of the function.
https://github.com/rikokko/to_be_frontend/pull/18/commits/3480ffdbdf8f705b6ed782a371fbc90d2946024f